README.md

```diff
@@ -1,13 +1,37 @@
-# mysql-container
-
-This docker container image builds off of the [MySQL 8.0 on Ubuntu 20.04 LTS](https://hub.docker.com/r/ubuntu/mysql)
-as the base image and further installs mysql-shell.
+# MySQL Server ROCK
+This repository contains the packaging metadata for creating a ROCK of MySQL built from the official ubuntu MySQL package from the Ubuntu repository and further installs mysql-shell.  For more information on ROCKs, visit the [rockcraft Github](https://github.com/canonical/rockcraft). 
 
 Built for use in the [MySQL k8s charm](https://github.com/canonical/mysql-k8s-operator).
 
-## Build
-
-```shell
+## Building the ROCK
+The steps outlined below are based on the assumption that you are building the ROCK with the latest LTS of Ubuntu.  If you are using another version of Ubuntu or another operating system, the process may be different.
 
-docker build -t mysql:latest .
+### Clone Repository
+```bash
+git clone git@github.com:canonical/charmed-mysql-container.git
+cd mongodb-rock
+```
+### Installing Prerequisites
+```bash
+sudo snap install rockcraft --edge
+sudo snap install docker
+sudo snap install lxd
+sudo snap install skopeo --edge --devmode
 ```
+### Configuring Prerequisites
+```bash
+sudo usermod -aG docker $USER 
+sudo lxd init --auto
+```
+*_NOTE:_* You will need to open a new shell for the group change to take effect (i.e. `su - $USER`)
```

Comment:
In that scenario, it would be a `newgrp - docker`, but that would automatically reload the session which may not be desirable.  But ya, its just there for the end user to know to reload their shell whether its with `su`, `newgrp`, or something else.